Formula/root.rb

```diff
@@ -62,7 +56,11 @@ def install
               "http://lcgpackages",
               "https://lcgpackages"
 
-    args = std_cmake_args + %W[
+    args = std_cmake_args
+
+    depends_on :xcode if MacOS.version <= :catalina
```

Comment:
This needs to be outside the `install` method along with the other dependencies.

Reply:
Oops, will fix. Not for this second, but should the other dependencies be Symbols too, or do they need to be strings?

Reply:
Formula dependencies are strings. Dependencies that are `Requirement`s (e.g. `:arch`, `:xcode`, etc) are `Symbol`s.